packages/core/contracts/financial-templates/long-short-pair/LongShortPair.sol

```diff
@@ -179,7 +179,9 @@ contract LongShortPair is Testable, Lockable {
      * @return collateralUsed total collateral used to mint the synthetics.
      */
     function create(uint256 tokensToCreate) public preExpiration() nonReentrant() returns (uint256 collateralUsed) {
-        collateralUsed = FixedPoint.Unsigned(tokensToCreate).mul(FixedPoint.Unsigned(collateralPerPair)).rawValue;
+        // Note the use of mulCeil to prevent small collateralPerPair causing rounding of collateralUsed to 0 enabling
+        // callers to mint dust LSP tokens without paying any collateral.
+        collateralUsed = FixedPoint.Unsigned(tokensToCreate).mulCeil(FixedPoint.Unsigned(collateralPerPair)).rawValue;
```

Comment:
No need to change the mul to mulCeil in redeem right? Because in all situations it's ok if the contract has more collateral than expected. I.e. no one can steal collateral

Reply:
Agree. I also looked through the settlement logic and it looks like it always rounds down too.